src/components/EditSections/EditContactInfo/EditContactInfo.test.js

```diff
@@ -76,4 +76,10 @@ describe('Render Edit contact Information component', () => {
     await userEvent.type(document.querySelector('[id="adduser_email"]'), 'Test@gmail.com');
     expect(document.querySelector('[id="adduser_email"]').value).toBe('Test@gmail.com');
   });
+  it('should render with disabled fields', () => {
+    renderEditContactInfo({ ...props, disabled: true });
+    expect(document.querySelector('[id="adduser_email"]')).toBeDisabled();
+    expect(document.querySelector('[id="adduser_phone"]')).toBeDisabled();
```

Comment:
Is this possible to use some getByRole or by label query instead of querySelector?